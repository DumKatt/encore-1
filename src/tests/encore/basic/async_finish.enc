class Main
  def main(): void {
    let f = async{print "Tasks are awesome"; "Outside finish"} in {
      finish {
	print 23;
        f
      };
      print "OUT";
      print get f
    }
  }
