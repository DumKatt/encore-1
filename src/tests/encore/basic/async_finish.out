23
OUT
Tasks are awesome
Outside finish
